src/store/create-store-module.js

```diff
@@ -52,7 +52,7 @@ export default (oidcSettings, storeSettings = {}, oidcEventListeners = {}) => {
   }
 
   const routeIsOidcCallback = (route) => {
-    if (route.meta.isOidcCallback) {
+    if (route.meta&&route.meta.isOidcCallback) {
```

Comment:
I have used standardJS in this project, operator && must be spaced ([space-infix-ops](https://eslint.org/docs/rules/space-infix-ops))

Since no linting is in place (I will add that soon) it is not super clear, but if you add spaces here this can be merged!